### DISCUSSION TITLE
502 bad gateway

### DISCUSSION THREAD

**Discussion by Elie-Saliba** [mozts2005/ZendeskApi_v2#566] [General]
HI sometimes I'm facing an issue while trying to use the CreateTicketAsync,
a 502 Bad Gateway Error is occurring.
After a few moments the same call is working. any idea what could be the issue , please find below the Error that is being returned.
your help will be Highly appreciated.

One or more errors occurred. (The remote server returned an error: (502) Bad Gateway.Error content: <html>
<head><title>502 Bad Gateway</title><script async src='/cdn-cgi/bm/cv/669835187/api.js'></script></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx</center>
<script type="text/javascript">(function(){window['__CF$cv$params']={r:'6f211cab587356e6',m:'uERLBIAn0Ni0i7x7NbysiyaasuTWUbiSg5NwfpddhGQ-1648310561-0-AQ/YlZw2yyB1NsMUiHmsNbMrHlsStbOm+yXOtbwe4pe7XmWVaPPzco0LdiMByXvhR9zOhMnNbY36uGqHnF0m/sUrMtKB/QCXKd9VH8ISFaOgaqiXdcoj6jP8/uwVJXmiUTqbwL8XJlENGZNXp2g7WnHXzmMM2Xq+XkkLdKQ5s2hpCCHwr62bwGQhTIsJLPxUFQ==',s:[0xc7d47a5153,0x3a0bf623fe],}})();</script></body>
</html>

Resource String: uploads.json?filename=executionReport.json +
File Name: executionReport.json
File Length: 16360
Date: Sat, 26 Mar 2022 16:02:41 GMT
Transfer-Encoding: chunked
Connection: keep-alive
Zendesk-Api-Version: 2022-01-01
X-Zendesk-Zorg: yes
X-Request-ID: 6f211cab587356e6-DUB
CF-Cache-Status: DYNAMIC
Expect-CT: max-age=604800, report-uri="[https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"](https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct%22)
Report-To: {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v3?s=%2FJl174zOhtJ25hZ8X9Iqe0sGxHV9b5z9S1vD80%2BMqgW%2B%2Blif%2F%2BoCwyaHzNNBttMU0hU52TNhSVDntIICYtFeVfSLB5JD%2FZs6N%2FBCTGVjpq4E43YBx3gLWqEWi31J7UL7%2FMJws6MyANk%3D"}],"group":"cf-nel","max_age":604800}
NEL: {"success_fraction":0.01,"report_to":"cf-nel","max_age":604800}
Set-Cookie: __cfruid=779fa7f5baef6643d2b4a3874ec405ab12fbd0da-1648310561; path=/; domain=.domain.zendesk.com; HttpOnly; Secure; SameSite=None
Server: cloudflare
CF-RAY: 6f211cab587356e6-IAD
Content-Type: text/html

)

**Comment by Elie-Saliba** [comment]
Hello, 
can I please get a response regarding the above.
Thank you

**Comment by mozts2005** [comment]
Please, let us know if you are still having this issue and we will try and help you out, but it looks like an issue out of the projects scope as we are a .NET library, and the issue is with JavaScript code in node.
